<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelImagePath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Путь к изображению:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelImageSource">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>149</y>
      <width>600</width>
      <height>391</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditImagePath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>581</width>
      <height>27</height>
     </rect>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSelectImage">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Обзор...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLoadImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить изображение</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderClusterCount">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>160</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>32</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>4</number>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSegmentation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Сегментировать
изображение</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelIterationCount">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>321</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSegmentCount">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Сегментов: 10&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>702</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
